api/Response.json

```diff
@@ -418,10 +418,10 @@
               }
             ],
             "safari": {
-              "version_added": false
+              "version_added": "10.1"
             },
             "safari_ios": {
-              "version_added": false
+              "version_added": "10.1"
```

Comment:
Double checking here that the mapping is right. Did you test both of these platforms? If just desktop, then it might be `10.3` for iOS, based on our browser engine data.

Reply:
Ugh, yeah, that sounds like the right thing to do, even though it's a bit more work. Seems pretty likely that the other data is wrong.